src/ripple/core/Pg.cpp

```diff
@@ -320,6 +324,13 @@ PgPool::PgPool(
     Stoppable& parent)
     : Stoppable("PgPool", parent), j_(j)
 {
+    // Make sure that boost::asio initializes the SSL library.
+    {
+        boost::asio::ssl::detail::openssl_init<true> initSsl;
```

Comment:
Could this be `static`?  That would initialize it once per application run, as opposed to every time a `PgPool` is constructed (which might be the same thing?).